README.md

```diff
@@ -1,130 +1,209 @@
 # Smartcar JS Client SDK [![Build Status][ci-image]][ci-url] [![GitHub tag][tag-image]][tag-url]
 
-The official Smartcar Javascript SDK.
+The official Smartcar JavaScript SDK
 
 ## Overview
-The [Smartcar API](https://smartcar.com/docs) lets you read vehicle data (location, odometer) and send commands to vehicles (lock, unlock) to connected vehicles using HTTP requests.
 
-To make requests to a vehicle from a web or mobile application, the end user must connect their vehicle using [Smartcar's authorization flow](https://smartcar.com/docs#authentication). The Smartcar Javascript SDK provides two scripts that assist with running Smartcar's authorization flow.
-- `sdk.js`: attaches to the browser's window as `Smartcar`. Allows you to generate a Smartcar OAuth URL and launch Smartcar's authorization flow in a pop-up.
-- `callback.js`: runs on load. This file is optional and can be served in the last page of your OAuth flow to close the popup window and call the callback function that was passed in when initializing `sdk.js`.
+Smartcar is the connected car [API](https://smartcar.com/docs) that allows mobile and web apps to communicate with connected vehicles across brands (think “check odometer” or “unlock doors.”)
+
+To make requests to a vehicle from a web application, the end user must connect their vehicle using [Smartcar's authorization flow](https://smartcar.com/docs#authentication). The Smartcar JavaScript SDK provides two scripts that assist with running this flow.
+
+- `sdk.js`: Sourced on the page on which your authorization flow begins. Attaches to the browser's window as `Smartcar`. Allows you to generate a Smartcar OAuth URL and to launch Smartcar's authorization flow in a pop-up.
+- `redirect.js`: Sourced on redirect page following completion of OAuth flow. Runs on load. This *optional* file can be served in the last page of your OAuth flow to close the pop-up window and trigger firing of an optional `onComplete` method passed to the `Smartcar` constructor from `sdk.js`. If you use a Smartcar-hosted redirect (explained in greater detail below), this file is automatically served and the `onComplete` method becomes required.
```

Comment:
instead of introducing the term "Smartcar-hosted redirect" immediately, we could say something along the lines of "this file can be sourced in a page served in your application or you can redirect to a version that is hosted by Smartcar (explained in greater detail below)"

I don't think we should  mention `onComplete` here